app/assets/javascripts/discourse/app/models/post.js

```diff
@@ -305,6 +305,12 @@ const Post = RestModel.extend({
   },
 
   toggleBookmark() {
+    let postEl = document.querySelector(`[data-post-id="${this.id}"]`);
```

Comment:
Not convinced that this is the cleanest way to do this...

Reply:
doesn't strike me as being terrible:) this is a simple and pragmatic solution...

Future might prove it ot be brittle, but we can advise if it's the case